item.html

```diff
@@ -357,38 +357,8 @@ <h2 class="ftco-heading-2">Have a Questions?</h2>
   
   form.date.min = `${yyyy}-${mm}-${dd}`;
   
-  // function writeNewOrder() {
-  //  var uid = sessionStorage.getItem('userUID');
-  //  // A post entry.
-  //  var orderData = {
```

Comment:
I think I did like the method of creating an object containing all data of the order rather than the current.